desktop-src/SbsCs/application-manifests.md

```diff
@@ -181,11 +181,16 @@ The **file** element has the attributes shown in the following table.
 
 ### activeCodePage
 
-Force a process to use UTF-8 as the process code page.
+On Windows 10, this element forces a process to use UTF-8 as the process code page. For more information, see [Use the UTF-8 code page](/windows/uwp/design/globalizing/use-utf8-code-page). On Windows 10, the only valid value for **activeCodePage** is **UTF-8**.
 
-**activeCodePage** was added in Windows Version 1903 (May 2019 Update). You can declare this property and target/run on earlier Windows builds, but you must handle legacy code page detection and conversion as usual. See [Use the UTF-8 code page](/windows/uwp/design/globalizing/use-utf8-code-page) for details.
+Starting in Windows 11, this element also allows selection of either the legacy non-UTF-8 code page, or code pages for a specific locale for legacy application compatibility. Modern applications are strongly encouraged to use Unicode. On Windows 11, this value may also be set to **Legacy** or a locale name such as **en-US** or **ja-JP**.
 
-This element has no attributes. **UTF-8** is only valid value for **activeCodePage** element.
+- On machines configured to a UTF-8 system active code page, **Legacy** will revert the process to the system locale code pages. If the system locale does not have defined code pages, then Windows-1252/437 will be used. The **Legacy** codepage setting is only supported in Fusion manifests and only beginning with Windows 11
```

Comment:
nit: period after Windows 11

Reply:
Thanks - I just fixed this in the internal docs repo, and the fix should be on the live docs site shortly.

Reply:
Thanks! 👍